consumers/notifiers/notifier.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Abstract Machines
+// SPDX-License-Identifier: Apache-2.0
+
+package notifiers
+
+import (
+	"errors"
+
+	"github.com/absmach/supermq/pkg/messaging"
+)
+
+// ErrNotify wraps sending notification errors.
+var ErrNotify = errors.New("error sending notification")
+
+// Notifier represents an API for sending notification.
+//
+//go:generate mockery --name Notifier --output=./mocks --filename notifier.go --quiet --note "Copyright (c) Abstract Machines"
+type Notifier interface {
```

Comment:
It's OK to keep the Notifier service (CRUD for Subscriptions) in Magistrala, but please move the Notifier interface to SuperMQ `/consumers` package.